plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountsPanel.java

```diff
@@ -0,0 +1,162 @@
+ *
+ * TODO(chanseok): move the panel to the bottom-right corner: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/740
+ */
+public class AccountsPanel extends PopupDialog {
+
+  private static final int DEFAULT_MARGIN = 5;
+  private static final int EMAIL_LEFT_MARGIN = 8;
+
+  private IGoogleLoginService loginService;
+
+  public AccountsPanel(Shell parent, IGoogleLoginService loginService) {
+    super(parent, SWT.MODELESS,
+        true /* takeFocusOnOpen */,
+        false /* persistSize */,
+        false /* persistLocation */,
+        false /* showDialogMenu */,
+        true /* showPersistActions */,
+        null /* no title area */, null /* no info text area */);
+    this.loginService = loginService;
+  }
+
+  @Override
+  protected Color getBackground() {
+    return getShell().getDisplay().getSystemColor(SWT.COLOR_WIDGET_BACKGROUND);
+  }
+
+  @Override
+  protected Color getForeground() {
+    return getShell().getDisplay().getSystemColor(SWT.COLOR_WIDGET_FOREGROUND);
+  }
+
+  @Override
+  protected Control createDialogArea(Composite parent) {
+    Composite container = (Composite) super.createDialogArea(parent);
+    GridLayoutFactory.swtDefaults().margins(DEFAULT_MARGIN, DEFAULT_MARGIN).applyTo(container);
+
+    createAccountsPane(container);
+    createButtons(container);
+    return container;
+  }
+
+  private void createAccountsPane(Composite container) {
+    Label messageLabel = new Label(container, SWT.NONE);
+    messageLabel.setText(Messages.LABEL_ACTIVE_ACCOUNT);
+    FontUtil.convertFontToBold(messageLabel);
+
+    Composite activeAccountIndenter = new Composite(container, SWT.NONE);
+    GridLayoutFactory.swtDefaults().margins(EMAIL_LEFT_MARGIN, 0).applyTo(activeAccountIndenter);
+
+    Account activeAccount = loginService.getActiveAccount();
+    Label accountLabel = new Label(activeAccountIndenter, SWT.NONE);
+    accountLabel.setText(activeAccount.getEmail());
+    FontUtil.convertFontToBold(accountLabel);
+
+    Set<Account> accounts = loginService.listAccounts();
+    if (accounts.size() > 1) {
+      new Label(container, SWT.NONE).setText(Messages.LABEL_OTHER_ACCOUNTS);
+
+      Composite accountsIndenter = new Composite(container, SWT.NONE);
+      GridLayoutFactory.swtDefaults().margins(EMAIL_LEFT_MARGIN, 0).applyTo(accountsIndenter);
+
+      for (Account account : accounts) {
+        if (!account.getEmail().equals(activeAccount.getEmail())) {
+          Link link = new Link(accountsIndenter, SWT.NO_FOCUS);
+          link.setText("<a href=\"" + account.getEmail() + "\">" + account.getEmail() + "</a>"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+          link.addSelectionListener(new SwitchAccountOnClick());
+        }
+      }
+    }
+  }
+
+  private void createButtons(Composite container) {
+    Composite buttonArea = new Composite(container, SWT.NONE);
+    GridDataFactory.defaultsFor(buttonArea).align(SWT.END, SWT.BEGINNING).applyTo(buttonArea);
```

Comment:
`GridDataFactory.fillDefaults().applyTo(buttonArea)` might achieve the same effect.